profile.yml

```diff
@@ -0,0 +1,85 @@
+---
+# All variables needed to deploy the tripleo-quickstart-scalelab project.
+
+rhos_release_rpm: "{{ lookup('env', 'RHOS_RELEASE_URL') }}"
+
+# OSP/OSPd versioning and build
+version: 11
+rhel_version: 7.4
+rhos_release: 11-director
+build: z4
+
+deploy_additional_repos: false
+
+repos:
+  rhel-7-server-beta:
+    baseurl: "{{ lookup('env', 'RHEL_7_SERVER_BETA_BASE_URL') }}"
+
+# Scale lab boot order still broke?
+scale_lab_broke: false
+
+# bulk introspection
+introspection: false
+
+# better introspection, be sure to turn bulk off if you use this
+introspect_with_retry: true
+
+# Dump version file to this directory
+version_directory: /etc
+
+# Stack user password. Encrypted with sha512
+stack_password: "{{ lookup('env', 'STACK_PASSWORD') }}"
+
+# undercloud control plane interface:
+local_interface: enp94s0f1
+
+# option to enable/disable ironic node cleaning on overcloud nodes
+node_cleaning: true
+
+overcloud_ssl_endpoints: false
+external_network_vip: "{{ lookup('env', 'EXTERNAL_NETWORK_VIP') }}"
+
+# External private vlan on undercloud:
+deploy_external_private_vlan: true
+external_vlan_device: enp94s0f1.10
+private_external_address: "{{ lookup('env', 'PRIVATE_EXTERNAL_ADDRESS') }}"
+private_external_netmask: 255.255.255.0
+# Tripleo maps external network port to noop.yml, keep the default behavior
+allow_external_on_compute: false
+
+# neutron dns:
+dns_server: "{{ lookup('env', 'DNS_SERVER') }}"
+
+# instackenv:
+instackenv_json: "{{ lookup('env', 'INSTACKENV_JSON') }}"
+
+cloud_title: "{{ lookup('env', 'CLOUD_TITLE') }}"
+ticket_number: "{{ lookup('env', 'TICKET_NUMBER') }}"
+foreman_url: "{{ lookup('env', 'FOREMAN_URL') }}"
+undercloud_hostname: "{{ lookup('env', 'UNDERCLOUD_HOSTNAME') }}"
+
+openstack_deployment_hosts:
```

Comment:
This bit here is going to be the hard part to do with environmental vars. Right now we have a static deployment and a scale up right?